lte/gateway/c/session_manager/sessiond_main.cpp

```diff
@@ -227,6 +235,16 @@ long get_quota_exhaust_termination_time(const YAML::Node& config) {
   return quota_exhaust_termination_on_init_ms;
 }
 
+sentry_config_t get_sentry_configuration(
+    magma::mconfig::SharedMconfig mconfig) {
```

Comment:
Maybe this function can be pulled into the sentry library

Reply:
This would also be affected by the refactoring envisioned and probably necessary to support the MME case, described above. So the take here was to create a minimal viable solution and see, how we can abstract with the next one.